README.md

```diff
@@ -173,46 +99,13 @@ logging config format for content details.
+If you are running into any issue, please check out [vDVS known issue page](http://vmware.github.io/docker-volume-vsphere/documentation/known-issues.html) related with vSphere docker volume service before raising a new issue.
 
 ## Contact us
+Please reach us out for any questions or problem you are facing with respect to vDVS. Please [click here](http://vmware.github.io/docker-volume-vsphere/documentation/contactus.html) to get contact detail.
+
+## Blogs
 
-### Public
-* [containers@vmware.com](containers@vmware.com)
-* [Issues](https://github.com/vmware/docker-volume-vsphere/issues)
-* [![Join the chat at https://gitter.im/vmware/docker-volume-vsphere](https://badges.gitter.im/vmware/docker-volume-vsphere.svg)](https://gitter.im/vmware/docker-volume-vsphere?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-* [VMware Docker Slack](https://vmwarecode.slack.com/messages/docker/) channel
-
-# Blogs
-
-- Cormac Hogan
-    - Overview
-        - [Docker Volume Driver for vSphere](http://cormachogan.com/2016/06/01/docker-volume-driver-vsphere/)
-        - [Docker Volume Driver for vSphere – short video](http://cormachogan.com/2016/06/03/docker-volume-driver-vsphere-short-video/)
-    - Docker + VSAN
-        - [Docker Volume Driver for vSphere on Virtual SAN](http://cormachogan.com/2016/06/09/docker-volume-driver-vsphere-virtual-san-vsan/)
-        - [Using vSphere docker volume driver to run Project Harbor on VSAN](http://cormachogan.com/2016/07/29/using-vsphere-docker-volume-driver-run-project-harbor-vsan/)
-        - [Docker Volume Driver for vSphere using policies on VSAN](http://cormachogan.com/2016/09/26/docker-volume-driver-vsphere-using-policies-vsan-short-video/)
-    - 0.7 Release Overview
-        - [Some nice enhancements to Docker Volume Driver for vSphere v0.7](http://cormachogan.com/2016/10/06/nice-enhancements-docker-volume-driver-vsphere-v0-7/)
-- William Lam
-    - [Getting Started with Tech Preview of Docker Volume Driver for vSphere - updated](http://www.virtuallyghetto.com/2016/05/getting-started-with-tech-preview-of-docker-volume-driver-for-vsphere.html)
-- Virtual Blocks
-    - [vSphere Docker Volume Driver Brings Benefits of vSphere Storage to Containers](https://blogs.vmware.com/virtualblocks/2016/06/20/vsphere-docker-volume-driver-brings-benefits-of-vsphere-storage-to-containers/)
-    - [vSphere Docker Volume Service is now Docker Certified!](https://blogs.vmware.com/virtualblocks/2017/03/29/vsphere-docker-volume-service-now-docker-certified/)
-- Ryan Kelly
-    - [How to use the Docker Volume Driver for vSphere with Photon OS](http://www.vmtocloud.com/how-to-use-the-docker-volume-driver-for-vsphere-with-photon-os/)
+List of blogs related to vSphere Docker Volume Service are found at [vDVS Blogs](http://vmware.github.io/docker-volume-vsphere/documentation/blogs.html).
```

Comment:
Let's just provide the link. No additional text is needed.

Reply:
done!